contracts/ligo/src/english_auction/english_auction_tez_permit.mligo

```diff
@@ -0,0 +1,100 @@
+#include "./english_auction_tez.mligo"
+
+type permit = 
+  [@layout:comb]
+  {
+    signerKey: key;
+    signature: signature;
+    paramHash: bytes;
+  }
+
+type permit_storage = 
+    [@layout:comb]
+  {
+    auction_storage : storage;
+    counter : nat;
+  }
+
+type permit_config_param =
+  [@layout:comb]
+  {
+    config : configure_param;
+    optional_permit : permit option;
+  } 
+
+type permit_auction_entrypoints = 
+  | Auction of auction_without_configure_entrypoints
+  | Permit_config of permit_config_param list
+
+type permit_return = operation list * permit_storage
+
+let address_from_key (key : key) : address =
+  let a = Tezos.address (Tezos.implicit_account (Crypto.hash_key key)) in
+  a
+
+let config_storage_with_permit (p, permit_storage : permit_config_param * permit_storage)  : permit_storage = begin
```

Comment:
Should check `fail_if_paused storage.pauseable_admin;`

Reply:
Yes addd back